CONTRIBUTING.md

```diff
@@ -19,6 +19,8 @@ Available via [https://github.com/equinix/terraform-provider-equinix/blob/main/.
 
 Please submit change requests and / or features via [Issues](https://github.com/equinix/terraform-provider-equinix/issues). There's no guarantee it'll be changed, but you never know until you try. We'll try to add comments as soon as possible, though.
 
+All resource and data source changes must be reflected in the documentation, including new attributes and changes to existing attribtues. The documentation is generated via `make docs`.  For more detailed instructions for updating docs, see ["Documenting the provider" in the development guide](./DEVELOPMENT.md#documenting-the-provider).
```

Comment:
"to existing attribtues" --> "to existing attributes"

Reply:
fixed